Cargo.toml

```diff
@@ -27,9 +27,10 @@ targets = [
 ]
 
 [features]
-default = [ "tray" ]
-tray = [ "libappindicator" ]
-ayatana = [ "libayatana-appindicator" ]
+default = [ "tray", "ayatana-appindicator" ]
```

Comment:
Can we still default to old libappindicator? The majority of most distros are still using it.